controllers/dspipeline_params.go

```diff
@@ -584,6 +585,7 @@ func (p *DSPAParams) ExtractParams(ctx context.Context, dsp *dspa.DataSciencePip
 	p.PiplinesCABundleMountPath = config.GetCABundleFileMountPath()
 	p.PodToPodTLS = false
 	p.FIPSEnabled = config.GetBoolConfigWithDefault(config.FIPSEnabledConfigName, config.DefaultFIPSEnabled)
+	p.WebhookName = "kubernetes-webhook"
```

Comment:
I think this may be clearer.
```suggestion
	p.WebhookName = "data-science-pipelines-webhook"
```